### ISSUE TITLE
failing CI

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/tripal_curator#23]
current master is failing CI: why?

**Comment by bradfordcondon** [comment]
https://travis-ci.org/statonlab/tripal_curator/builds/410987995?utm_source=slack&utm_medium=notification

i think just replacing the set-up with factories would go a long way.

**Comment by bradfordcondon** [comment]
* Tests past locally
* [ fail remote](https://travis-ci.org/statonlab/tripal_curator/builds/410987995)

```

1) Tests\CValueTest::test_assign_value_to_cvalues_name
Exception: No Cvterm_id to set the property value to!
/var/www/html/sites/all/modules/custom/tripal_curator/includes/CValue.php:118
/var/www/html/sites/all/modules/custom/tripal_curator/tests/CValueTest.php:221
```


```

2) Tests\ChadoPropertyTest::test_initialize_property
PDOException: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "type_id" violates not-null constraint
DETAIL:  Failing row contains (7, 7, null, No cvalue!, 0, null).
/var/www/html/includes/database/database.inc:2227
/var/www/html/includes/database/pgsql/database.inc:106
/var/www/html/includes/database/pgsql/query.inc:96
/var/www/html/sites/all/modules/custom/tripal_curator/tests/ChadoPropertyTest.php:66
```


and then, the failures:

```

There were 5 failures:
1) Tests\CVTermTest::test_initialize_property
Failed asserting that null matches expected 'Curator Test'.
/var/www/html/sites/all/modules/custom/tripal_curator/tests/CVTermTest.php:50
2) Tests\CVTest::test_CV_inits_and_gets_terms
Failed asserting that an array is not empty.
/var/www/html/sites/all/modules/custom/tripal_curator/tests/CVTest.php:32
3) Tests\CVTest::test_CV_returns_prop_tables_on_init
The get_prop_tables() method failed to return property tables containing the CV (cvterm_property_type).
Failed asserting that an array is not empty.
/var/www/html/sites/all/modules/custom/tripal_curator/tests/CVTest.php:44
4) Tests\CValueTest::test_defining_by_cvalue_id
Failed asserting that an array is not empty.
/var/www/html/sites/all/modules/custom/tripal_curator/tests/CValueTest.php:157
5) Tests\CValueTest::test_assign_cvalue_id
Failed asserting that null is not null.
/var/www/html/sites/all/modules/custom/tripal_curator/tests/CValueTest.php:190
```

**Comment by bradfordcondon** [comment]
resolved #31
